client/components/mma/upgrade/UpgradeSupportAmountForm.tsx

```diff
@@ -232,27 +230,29 @@ export const UpgradeSupportAmountForm = ({
 							threshold={threshold}
 						/>
 					)}
-					{!continuedToConfirmation && !errorMessage && !!chosenAmount && (
-						<section css={buttonContainerCss}>
-							<ThemeProvider
-								theme={buttonThemeReaderRevenueBrand}
-							>
-								<Button
-									cssOverrides={buttonCentredCss}
-									onClick={() => {
-										setContinuedToConfirmation(
-											chosenAmount ? true : false,
-										);
-										scrollToConfirmChange();
-									}}
+					{!continuedToConfirmation &&
+						!errorMessage &&
+						!!chosenAmount && (
+							<section css={buttonContainerCss}>
+								<ThemeProvider
+									theme={buttonThemeReaderRevenueBrand}
 								>
-									Continue with {currencySymbol}
-									{formatAmount(chosenAmount)}/
-									{mainPlan.billingPeriod}
-								</Button>
-							</ThemeProvider>
-						</section>
-					)}
+									<Button
+										cssOverrides={buttonCentredCss}
+										onClick={() => {
+											setContinuedToConfirmation(
+												chosenAmount ? true : false,
```

Comment:
I know this wasn't changed here but not sure this makes sense anymore. Why would chosenAmount be falsy if we click? 🤔